src/components/ManageProject/ManageProjectGeneralSettingsPage.tsx

```diff
@@ -39,6 +48,28 @@ const ProjectGeneralSettingsFormSchema = z.object({
 
 export const ManageProjectGeneralSettingsPage = () => {
   const projectData = useJbProject()
+  const contractWrite = useJbProjectsSetMetadataOf()
```

Comment:
I realised half way thru that theres not really much point have a juicehook that wraps this whole thing. because, in theory a client could use different ipfs gateways which return different responses. +, all of them should be going thru an internal backend to not expose secretz.

Ends up not being too much code to have ehre anyways.